### PR TITLE
Implemented DELETE operator for REST.

### PR DESCRIPTION
**Pull request by dkeeney**
This extends the REST interface to include support for the DELETE operator.
Allows the REST app to delete a region, a link, or the entire Network object.

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\setup.py",
+  "PreviewInSolutionExplorer": false
```
Comment (breznak) [inline .vs/VSWorkspaceState.json:6]: did you mean to check-in these? esp the sqlite file below?

**Comment by dkeeney** [comment]
> did you mean to check-in these? esp the sqlite file below?

No, must be something new with VS2019
Let me add them to the .gitignore

**Comment by dkeeney** [comment]
Ok, trying again.  Lets make sure files from .vs are not included.
...good catch by the way. 👍

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (breznak) [reply .vs/VSWorkspaceState.json:6]: still there, you have to call `git rm -rf .vs ` and commit

**Comment by dkeeney** [comment]
Thank you

**Pull request merged**